static/clock.html

```diff
@@ -51,6 +52,9 @@ <h1 class="bgnow"></h1>
       var rec = xhr[0];
       var last = new Date(rec.date);
       var now = new Date( );
+      if (window.serverSettings.units == 'mmol') {
+        rec.sgv = (Math.round((rec.sgv / 18) * 10) / 10).toFixed(1);
```

Comment:
the fact that you need to move decimals around to round in js is mindblowing. I had to look it up because I didnt believe it. Since mmol conversions just like this probably happen frequently, might be nice to just have a class/function/whatever it is in js to simply take an sgv and return its mmol value, but this looks like you were just going for a quick fix so seems fine!